tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 envlist=
     py{36,37,38,39,310,py3}-core
-    lint
+    py{36,37,38,39,310}-lint
```

Comment:
It doesn't look like these are set up to correctly in the file. We could do a `common-lint` and extend from there for each. This is untested, but I think this should work:

```ini
[common-lint]
extras=lint
commands=
    mypy -p hexbytes --config-file {toxinidir}/mypy.ini
    flake8 {toxinidir}/hexbytes {toxinidir}/tests
    isort --recursive --check-only --diff {toxinidir}/hexbytes {toxinidir}/tests
    pydocstyle --explain {toxinidir}/hexbytes {toxinidir}/tests

[testenv:py{36,37,38,39,310}-lint]
extras={[common-lint]extras}
commands={[common-lint]commands}
```

Reply:
After updating the setup for the lint jobs, there should be some linting errors to fix as well.

Reply:
Actually... if we update `tox`, we could reduce the above to:

```ini
[common-lint]
extras=lint
commands=
    mypy -p hexbytes --config-file {toxinidir}/mypy.ini
    flake8 {toxinidir}/hexbytes {toxinidir}/tests
    isort --recursive --check-only --diff {toxinidir}/hexbytes {toxinidir}/tests
    pydocstyle --explain {toxinidir}/hexbytes {toxinidir}/tests

[testenv:py{36,37,38,39,310}-lint]
extras={[common-lint]extras}
commands={[common-lint]commands}

[testenv:lint]
basepython=python
extras={[common-lint]extras}
commands={[common-lint]commands}

```